src/products/firefox_ios/testing.md

```diff
@@ -23,10 +23,6 @@ Here is a list of issues you should try to identify when testing the build:
 
 Concerning untranslated content: please note that we currently have two tiers of support in iOS. If your locale is only in the Tier 2 support list, then it sometimes happens that menu items, which are part of the OS itself, may not be localizable - and will therefore unfortunately appear in English on the final UI. To tell if a locale is supported or not, simply take a look on an actual device, under the Language Settings of the most recent iOS version available. Tier 1 locales are the ones listed under “iPod/iPhone/iPad Language”, and Tier 2 locales appear under “Other Language” (or “Add Language” if there are already some languages under that list).
 
-## Testing a brand new locale
+## Shipping a brand new locale
 
-When a locale is brand new, l10n-drivers and the localization team will work through Bugzilla to process sign-offs on l10n testing.
-
-The l10n Mobile Project Manager will open up a meta bug for this and attach all needed bugs under their corresponding component. Closing the bug as resolved-fixed will signal that a locale is signing off on testing, and good to go for release.
-
-Details concerning all this process are always announced on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n).
+The locale is tested and has reached 20% completion, we will add the locale to release builds.
```

Comment:
Only 20%?

Reply:
Yes, this has been the case for a long time now. We agreed with mobile teams at one point that any new locale that gets to 20% can ship on iOS (this is not the case for android)